project_script/main.py

```diff
@@ -104,8 +102,49 @@ def integrate_data():
 
 density_of_carriers = [element * CHARGE_DENSITY_CALCULATION for element in integration_results]
 
-pd.DataFrame(density_of_carriers).to_csv(
-    "C:/Users/robee/Desktop/density_of_carriers_test.txt",
-    sep='\t',
-    decimal=','
-    )
+# pd.DataFrame(density_of_carriers).to_csv(
+#     "C:/Users/robee/Desktop/density_of_carriers_test.txt",
+#     sep='\t',
+#     decimal=',',
+#     )
+
+odd_columns_subtracted_smoothed_transposed = np.array(dt.smooth(odd_columns_subtracted_transposed_in_arrays))
+
+peaks = dt.find_the_peaks(odd_columns_subtracted_smoothed_transposed)
+
+indexes_list = dt.find_peak_index(odd_columns_subtracted_smoothed_transposed, peaks)
+
+time_max_values = dt.flatten(dt.find_time_max(time_photo_celiv_ramp_time_transposed, indexes_list))
+
+peak_values = pd.DataFrame([peaks, time_max_values]).transpose().set_axis(['current peak', 'peak time'], axis=1)
```

Comment:
The same of some comments above, three chaining or more, would be more readable if you break.

```
peak_values = pd.DataFrame([peaks, time_max_values])
    .transpose()
    .set_axis(['current peak', 'peak time'], axis=1)
```